build.sbt

```diff
@@ -52,6 +60,7 @@ lazy val clientLibrary = project // no need to define file, because path is same
     name := "login-service-client-library",
     libraryDependencies ++= clientLibDependencies,
     javacOptions ++= commonJavacOptions,
+    crossScalaVersions := Seq(scala212, "2.13.12")
```

Comment:
Hi, so I tested a local release and it does indeed publish the 2.13 version. I just haven't made a release tag yet since I'm waiting for this to be reviewed and merged before I do. `ci-release` uses `+publish` which publishes all the crossbuild versions. So no worries there. 

Will have a look at testing.

Reply:
> Hi, so I tested a local release and it does indeed publish the 2.13 version. I just haven't made a release tag yet since I'm waiting for this to be reviewed and merged before I do. ci-release uses +publish which publishes all the crossbuild versions. So no worries there.

Oh, that's great, I missed that.